### DISCUSSION TITLE
Custom themes / colors

### DISCUSSION THREAD

**Discussion by lawand-dot-io** [lawand-dot-io/taskbar#30] [Feature requests]
Add support for picking Taskbar color and text color etc, for example let the user pick the toolbar color like pure black etc

(Feature request split from https://github.com/lawand-dot-io/taskbar/discussions/20)

**Comment by Supervisor360** [comment]
Exactly! With my current desktop's theme, I love using taskbar but the problem is the colors of it does not really match the rest of the desktop. It would b great to configure the design like u can with Mate/Gnome2 panels!

**Comment by dhry** [comment]
PLEASE consider adding the ability to at least change the default background color of Taskbar. Black text on pure white would be my preference, the black on grey that it is now is just not easy enough for me to see without squinting. User-selectable font would be excellent as well.

Cheers
Dhry

**Reply by lawand-dot-io** [reply]
I see what you mean. unfortunately there's always something with higher priority. in the meantime, have you tried setting Taskbar theme to dark mode? that should be more readable.. also try changing Taskbar size and font size. how about that?

**Reply by dhry** [reply]
All done already. I just wanted to make sure that I threw my hat in on this feature request so you know how many people are interested in it. It's equally a matter of aesthetics as of readability.

Dhry

**Reply by lawand-dot-io** [reply]
Ok thanks for the info

**Comment by a-riddle-fan** [comment]
That would take your great application to the next level. Ideally, there would be a color selection (for the text and background) or the option to make the background transparent. So, for example, you could make the text white and the background transparent.
In any case, thank you very much for the software!

**Reply by lawand-dot-io** [reply]
@a-riddle-fan thanks for the input, I will keep this in mind. One of the upcoming next versions of the app will include a transparency option where you can make the taskbar more transparent so it would have the color of the wallpaper behind it. I think this will already be a good improvement, and perhaps in the future I would support custom colors etc.
